### DISCUSSION TITLE
Suggestion: use vite fable template intead of regular fable

### DISCUSSION THREAD

**Discussion by jkone27** [Lanayx/Oxpecker#38] [Ideas]
vite fable plugin has several advantages over regular fable, for example, no intermediate code has to appear in the repository, is faster, and unit tests can work great with vitest and also have HMR, also i think is a nice way to support a great fable community project?

* https://vitest.dev/
* https://nojaf.com/vite-plugin-fable/

maybe nice for a POC on a test branch

**Comment by Lanayx** [comment]
@jkone27 Thanks for suggesting, I saw this library before, however it's officially unsupported, so I can't recommend that way to people. But you still can create your own repository with the example of using that plugin and I'll add a reference to it in docs.

**Comment by fradav** [comment]
@jkone27 did you succeed at running solidjs and fable vite plugins together ? I tried but the fable compiler hangs

**Reply by jkone27** [reply]
Didn't try yet. If I have some time I will experiment too this week
